### DISCUSSION TITLE
窗体上确认和取消按钮，响应click事件

### DISCUSSION THREAD

**Discussion by arit1988** [BYJRK/DotNet-Discussions#84] [MVVM]
1 关闭窗体前，需要先调用viewmodel中的一些方法
2 调用window的close方法关闭窗体
现在都是写在了后置代码中，这种情况理想的方式该如何处理呢？

**Comment by CodingOctocat** [comment]
没必要过分追求 MVVM，在 `xaml.cs` 事件方法里调用 `Vm.Method()` 就行了。不要变得 MVVM 的优势没用到，反而带来一堆累赘。

**Comment by BYJRK** [comment]
用行为，Interaction.Triggers 注册窗口的 Closed 事件，然后绑定到 VM 中的 RelayCommand
